src/layouting/layouting.jl

```diff
@@ -135,6 +136,14 @@ function glyph_collection(
                     push!(lineinfos, view(charinfos, last_line_start:i))
                 end
 
+                if ci.char == '\n'
+                    push!(xs, Float32[])
+                    push!(lineinfos, view(charinfos, last_line_start:i))
+                    last_space_local_idx = 0
+                    last_line_start = i+1
+                    x = 0f0
+                end
```

Comment:
I think we could just make the `elseif ci.char == '\n' || i == length(charinfos)` an `if ci.char == '\n' || i == length(charinfos)` instead of repeating this